outputs_emissions.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: index
+title: Outputs from Emissions Modeling
+prev: emissions.html
+next: diagram.html
+gcam-version: v5.3 
+---
+
+## Description of Outputs
+
+| Name | Resolution | Unit | Query | XML Tag |
+| :--- | :--- | :--- | :--- | :--- |
+| Emissions<sup>[1,2](#table_footnote)</sup> |  Technology, Region<sup>[3](#table_footnote)</sup>, and Year | Various<sup>[4](#table_footnote)</sup>  | <span id="nonCO2 emissions by tech"><button onclick='getQuery("nonCO2 emissions by tech", "nonCO2 emissions by tech")'>nonCO2 emissions by tech</button></span> | `emissions` |
```

Comment:
Looks like "emissions-sequestered" and "indirect-emissions" belong here as well.

Reply:
I've added "emissions-sequestered". For "indirect-emissions", I need to check a bit more, but I don't think that is actively used anymore. It looks like it should always be zero, but I want to confirm.